cg/scripts/blockchain-function-test.js

```diff
@@ -1,27 +1,89 @@
-const {
-  getRectificationCount,
-  getIsErased,
-  getSubjectDataState,
-  getProcessors,
-  setSubjectDataState,
-  setProcessors,
-  isProcessor,
-  areAllValidProcessors,
-  recordProcessorsUpdate,
-  recordConsentGivenTo,
-  recordAccessByController,
-  recordRectificationByController,
-  recordErasureByController,
-  recordErasureByProcessor,
-  getPastEvents
-} = require('../src/utils/blockchain');
-
-getRectificationCount('0x0000000000000000000000000000000000000000000000000000000000000001')
-  .then(console.log)
-  .catch(console.error)
-  .then(() => process.exit(0));
-
-// recordRectificationByController('0x0000000000000000000000000000000000000000000000000000000000000001')
+// docker-compose exec cg node scripts/blockchain-function-test.js
```

Comment:
The test script is complete with the function calls now. This is good to know how to call each function and test it

Reply:
Yeah, I think it makes sense to keep this.